From b75353009778260be8f28d746b1c1bfc145e8d28 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 19 May 2021 18:55:02 +0300
Subject: [PATCH] [SW-0] module_pci: add notification about mem_en

Log PCIe MEM_EN attribute change.
This is helpful for understanding the events seen by PCIe.

Change-Id: Ie8a40090ad54fe89e7ab5ccd07f67dee4b809b5c
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot/module_pci.c b/preboot/module_pci.c
index 16b09b4..e108fff 100644
--- a/preboot/module_pci.c
+++ b/preboot/module_pci.c
@@ -127,6 +127,10 @@
 		rst_src = regmap_readl(mmPSOC_GLOBAL_CONF_RST_SRC);
 		if (pci_sts.mem_enabled && (rst_src < HL_WD_RST))
 			hl_pci_iatu_init();
+
+		preboot_puts_raw("PCIe mem_en changed to: ");
+		preboot_puts_raw(pci_sts.mem_enabled ? "true" : "false");
+		preboot_puts("\n");
 	}
 }
 
